configserver/src/main/resources/bootstrap.yml

```diff
@@ -3,18 +3,15 @@ spring:
     name: config-server
   profiles:
     active:
-    - git
+    - native
```

Comment:
I think we should keep the `git` profile we have as its more robust than using the classpath config..